lib/linux_admin.rb

```diff
@@ -40,11 +40,6 @@ class << self
     extend Forwardable
```

Comment:
Do we still need `Forwardable` and `Gem::Deprectate`?

Reply:
works - thanks